Jenkinsfile

```diff
@@ -37,9 +60,24 @@ pipeline {
 	}
 	post {
 		success {
+			script {
```

Comment:
@willclarktech I have a question about this section - the way I've done it so far, this will only post a success notification to Slack if it's preceded by a failure.

We do that I think for lisk hub, but lisk core I think reports success on every success. What do we want the behavior here to be?

Reply:
@Nazgolze What's the idea behind the hub approach? I'm probably the wrong person to ask as I don't like slack notifications for this kind of thing anyway.

Reply:
@slaweet - What's the idea behind the hub approach?